R/install_reqs.R

```diff
@@ -68,12 +68,24 @@ install_cran_package = function(package, version, repo, dryrun){
   return(failures)
 }
 
+identify_comparison_op = function(req){
+  comp_match = vapply(c(COMPS,'='),
+                      grepl,
+                      x = req,
+                      fixed = TRUE,
+                      FUN.VALUE = logical(1))
+  if(any(comp_match)){
+    comp = names(which.max(comp_match))
+  } else {
+    comp = NA_character_
+  }
+  return(comp)
```

Comment:
Because COMPS has been sorted by nchar when it was created and '=' is added to the end, it always matches exactly one. I added the catch for no match so I could use it in more places than just `process_versioned_requirement`